docs/build/building-flutter-applications.md

```diff
@@ -97,14 +96,14 @@ The next step in the build configuration is Signing. Here, please select the pro
 
 For signing iOS apps, press add, select the bundle ID from the first dropdown and then select a compatible provisioning profile (added from the signing identities module) from the second dropdown.
 
-<NarrowImage src="https://cdn.appcircle.io/docs/assets/image%20(5).png" />
+<Screenshot src="https://cdn.appcircle.io/docs/assets/build-profile-ios-signing-configuration.png" />
```

Comment:
This page has problems for "NarrowImage" to "Screenshot" conversion. :point_down: 

![Screenshot from 2023-12-01 07-03-15](https://github.com/appcircleio/appcircle-docusaurus/assets/7355362/4620ba58-e83f-4284-abb1-5d1841ca8ee9)

I'm fixing the cases here... :wrench: